data/StartRoomData.py

```diff
@@ -67,14 +82,22 @@ class StartRoomData:
         },
     ]
     )]),
-    RoomName.Warrior_Shrine.value: StartRoomData(area=MetroidPrimeArea.Magmoor_Caverns, loadouts=[
-        StartRoomLoadout([SuitUpgrade.Varia_Suit], [
-            {
-                "Magmoor Caverns: Warrior Shrine": [SuitUpgrade.Morph_Ball],
-                "Magmoor Caverns: Storage Cavern": [SuitUpgrade.Morph_Ball_Bomb, SuitUpgrade.Main_Power_Bomb],
+    RoomName.Warrior_Shrine.value: StartRoomData(
+        area=MetroidPrimeArea.Magmoor_Caverns,
+        loadouts=[
+            StartRoomLoadout([SuitUpgrade.Varia_Suit], [
+                {
+                    "Magmoor Caverns: Warrior Shrine": [SuitUpgrade.Morph_Ball],
+                    "Magmoor Caverns: Storage Cavern": [SuitUpgrade.Morph_Ball_Bomb, SuitUpgrade.Main_Power_Bomb],
+                }
+            ]),
+        ],
+        allowed_elevators={
```

Comment:
Certain start room + elevator combos are not feasible due to lack of locations. I added both an `allow` and `deny` option for these scenarios depending on how many were feasible or not.